app/views/layouts/application.html.erb

```diff
@@ -18,12 +18,6 @@
           integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" 
           crossorigin="anonymous">
 
-
-    <!-- Font Awesome -->
-    <script src="https://kit.fontawesome.com/<%= Rails.application.credentials.dig(:font_awesome_download_key)%>.js"
```

Comment:
Yes, I didn't want to mix this ticket up with updating data in Keybase, so I've created a ticket addressing that - https://www.pivotaltracker.com/story/show/184115185

Reply:
We'll, it's about changes introduces in this story, so it's hardly mixing up. It's part of this story

Reply:
I didn't do it also because of that migrating to npm font awesome may not have been accepted and in that scenario changes to the Keybase would have to be reverted.